react/components/ProductBrand/urlHelpers.js

```diff
@@ -0,0 +1,56 @@
+export const DEFAULT_WIDTH = 'auto'
```

Comment:
This file has been copied from somewhere else (and from the looks of it, has been replicated several times already)

This is supposed to be temporary—a smart `VTEXImage` component should be created in the near future that handles this. Just needs its interface to be ironed out.